scripts/monit_start_service.py

```diff
@@ -26,19 +28,31 @@
 # The default port to connect to Cassandra.
 CASSANDRA_PORT = 9999
 
+# The service name for monit to start Cassandra.
+CASSANDRA_SERVICE_NAME = "cassandra"
```

Comment:
Are those constants also defined somewhere else? Can you use the existing ones if any?